### PR TITLE
* add tracking params

### PR DESCRIPTION
**Pull request by jayzeng**

### CODE REVIEW

File: src/knock-integration.js [hunk 1]
```diff
@@ -65,9 +65,9 @@
...
 
                 if (listingId){
-                    url += '/listing/'+listingId;
+                    url += '/listing/'+listingId+'?s=w';
                 } else {
-                    url += '/company/'+this.companyId;
+                    url += '/company/'+this.companyId+'?s=w';
                 }
 
                 url += '?isExternal=true&companyName='+this.companyId;
```
Comment (kieferhagin) [inline src/knock-integration.js]: we already append query params here

Comment (kieferhagin) [inline src/knock-integration.js]: url += '?isExternal=true&companyName='+this.companyId+'&s=w';

**Comment by jayzeng** [comment]
synced

File: src/knock-integration.js [hunk 2]
```diff
@@ -70,7 +70,7 @@
                     url += '/company/'+this.companyId;
                 }
 
-                url += '?isExternal=true&companyName='+this.companyId;
+                url += '?isExternal=true&companyName='+this.companyId+'?s=w';
```
Comment (kieferhagin) [inline src/knock-integration.js]: jay, you're killing me brah....

`?s=w` should be `&s=w`

Comment (jayzeng) [inline src/knock-integration.js]: errr....need some pills

**Comment by jayzeng** [comment]
here we go

File: src/knock-integration.js (hunk 2 above)
Comment (kieferhagin) [inline src/knock-integration.js]: haha i feel your pain. my head is splittin

**Pull request merged**